components/BottomNavigation/tests/snapshot/MDCBottomNavigationBarControllerSnapshotTests.m

```diff
@@ -0,0 +1,222 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialSnapshot.h"
+
+#import "MDCBottomNavigationBarController.h"
+
+/** Number of cells shown in the scrollable child view controller. */
+static const CGFloat kNumberOfCellsInScrollableChild = 25;
+
+/** The size of a cell in the scrollable child view controller. */
+static const CGSize kSizeOfCellInScrollableChild = (CGSize){(CGFloat)48, (CGFloat)48 };
+
+/** A view controller with content that expands to fit its bounds. */
+@interface MDCBottomNavigationBarControllerSnapshotTestFixedContentChildViewController : UIViewController
+@end
+
+@implementation MDCBottomNavigationBarControllerSnapshotTestFixedContentChildViewController
+@end
+
+/** A view controller with content that scrolls. */
+@interface MDCBottomNavigationBarControllerSnapshotTestScrollableChildViewController : UICollectionViewController <UICollectionViewDataSource>
+/** The flow layout for this view controller. */
+@property(nonatomic, readonly, nonnull) UICollectionViewFlowLayout *flowLayout;
+@end
+
+@implementation MDCBottomNavigationBarControllerSnapshotTestScrollableChildViewController
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  [self.collectionView registerClass:[UICollectionViewCell class] forCellWithReuseIdentifier:@"cell"];
+  self.collectionView.dataSource = self;
+  if (@available(iOS 11.0, *)) {
+    self.collectionView.insetsLayoutMarginsFromSafeArea = YES;
+    self.collectionView.contentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentAlways;
+  }
+}
+
+- (UICollectionViewFlowLayout *)flowLayout {
+  return (UICollectionViewFlowLayout *)self.collectionViewLayout;
+}
+
+- (NSInteger)numberOfSectionsInCollectionView:(UICollectionView *)collectionView {
+  return 1;
+}
+
+- (NSInteger)collectionView:(UICollectionView *)collectionView numberOfItemsInSection:(NSInteger)section {
+  return 25;
```

Comment:
why is this not linked to the const on top of the file

Reply:
Done.